### PR TITLE
Camilla Hallberg & Fay Pistikozoglou (Week 4 - Chatbot Project)

### PR DESCRIPTION
**Pull request by faypi**
Worked together with @CamillaHallberg  and create a chatbot in JS.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,24 @@
...
 
 Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
 
 ## View it live
 
-Have you deployed your project somewhere? Be sure to include the link to the deployed project so that the viewer can click around and see what it's all about.
+https://camilla-fay-chatbot.netlify.app/
+
+
+## Planning for the Bot
```
Comment (Sherin-Susan-Thomas) [inline README.md:14]: Its good that you included this part. Its insightful :)

File: code/index.html [hunk 2]
```diff
@@ -1,33 +1,55 @@
...
             Send
           </button>
         </form>
       </div>
     </main>
+  </div>
+  <div class="footer">
+    <div class="footer-left">
+      <p>
+        Made with └[∵┌]└[ ∵ ]┘[┐∵]┘ in SE and GR
```
Comment (idanaslund) [inline code/index.html:37]: Fun!

File: code/script.js [hunk 3]
```diff
@@ -30,18 +38,142 @@ const showMessage = (message, sender) => {
...
+  // setTimeout(() => selectDestination(name), 1000)  //pass arguments to selectDestination function with 1s delay --- update: no need to pass argument since we store it in global var
+}
+
+// Question 2
+const askQuestion2 = () => {
+  showMessage(`Where do you want to be picked up from ${username} ?`, 'bot') //show next quesiton from bot
+  inputWrapper.innerHTML = //create a new form - the previous form is deleted ALSO added autofocus to the input so the cursor goes to the input and the user doesn't have to click to type an answer
+    ` 
+    <form id="address-form">
+      <input id="address-input" type="text" autofocus />
```
Comment (idanaslund) [inline code/script.js:64]: Very good with autofocus, also great explanation in your comment. 
// Ida & Sherin

File: code/script.js [hunk 4]
```diff
@@ -30,18 +38,142 @@ const showMessage = (message, sender) => {
...
+  inputWrapper.innerHTML = ""
+  
+}
+// //REPLY 2
+// const areaOfDestination = (area) => {
+//   console.log(area)
+//   showMessage(area, 'user')
+// }
+// Set up your eventlisteners here
+form.addEventListener('submit', handleQuestion1) //This is the first event listener. Code will not run until it gets to that line. After that, it goes to the top and runs the handleQuestion1 function
```
Comment (idanaslund) [inline code/script.js:179]: Clever to restart the chat and very clear what is going on. 
// Ida & Sherin

File: code/script.js [hunk 5]
```diff
@@ -1,8 +1,14 @@
 // All the DOM selectors stored as short variables
 const chat = document.getElementById('chat')
+const form = document.getElementById('name-form')
+const inputWrapper = document.getElementById('input-wrapper')
+const sendBtn = document.getElementById('send')
 
 // Global variables, if you need any, declared here
-
```
Comment (Sherin-Susan-Thomas) [inline code/script.js:5]: Good choice of assigning Global variables :)

Ida & Sherin

File: code/style.css [hunk 6]
```diff
@@ -60,12 +57,21 @@ main {
...
+  background-color: #F6E1E2;
+}
+
 
 .chat {
   flex: 1;
   display: flex;
   justify-content: flex-start;
-  overflow: scroll;
+  /* changed overflow:scroll (code that was given to us) to overflow:auto in order to remove the scroll when not needed */
```
Comment (Sherin-Susan-Thomas) [inline code/style.css:73]: Good Choice with scroll. It looks classy to add this.

Ida & Sherin

File: code/style.css [hunk 7]
```diff
@@ -115,26 +121,147 @@ main {
...
+.footer a {
+  text-decoration: underline;
+  color: inherit;
+}
+
+.footer p {
+  text-align: center;
+}
+
+/* Responsiveness tablets and a bit smaller screens */
```
Comment (idanaslund) [inline code/style.css:200]: Good thing you remember the media queries!
// Ida & Sherin

**Review by Sherin-Susan-Thomas — APPROVED** [review]

**Review by idanaslund — APPROVED** [review]
It was easy to follow the code with your clear comments to the functions etc. in the JS. The project fulfills the general requirements, the blue level and also red level requirements. We learned the meaning of autofocus and we really learned the global variables from your code.  It looks nice with a clean design. Great job Fay and Camilla!

Reviews made by Ida and @Sherin-Susan-Thomas

**Comment by CamillaHallberg** [comment]
Thank you Ida and Sherin for your thoughts on our project :)

**Pull request closed without merging**